frontend/public/components/cluster-settings/cluster-settings.tsx

```diff
@@ -872,35 +926,35 @@ export const ClusterVersionDetailsTable: React.FC<ClusterVersionDetailsTableProp
   );
 };
 
-export const ClusterOperatorTabPage: React.SFC<ClusterOperatorTabPageProps> = ({ obj: cv }) => (
+export const ClusterOperatorTabPage: React.FC<ClusterOperatorTabPageProps> = ({ obj: cv }) => (
   <ClusterOperatorPage cv={cv} autoFocus={false} showTitle={false} />
 );
 
-const pages = [
-  {
-    href: '',
-    name: 'Details',
-    component: ClusterVersionDetailsTable,
-  },
-  {
-    href: 'clusteroperators',
-    name: 'Cluster Operators',
-    component: ClusterOperatorTabPage,
-  },
-  {
-    href: 'globalconfig',
-    name: 'Global Configuration',
-    component: GlobalConfigPage,
-  },
-];
-
-export const ClusterSettingsPage: React.SFC<ClusterSettingsPageProps> = ({ match }) => {
-  const title = 'Cluster Settings';
+export const ClusterSettingsPage: React.FC<ClusterSettingsPageProps> = ({ match }) => {
+  const { t } = useTranslation();
+  const title = t('cluster-settings~Cluster Settings');
```

Comment:
Cluster settings? (Here and in the nav?)

Reply:
Page titles are supposed to be title case per http://openshift.github.io/openshift-origin-design/conventions/documentation/capitalization.html#title-case